### ISSUE TITLE
Ensure links are distinguished by more than just color

### ISSUE THREAD

**Issue by eons44** [web-infrastructure/doc_architecture#30]
Links must be appear visually distinct from regular text. A common mistake is to make links only differ subtly in color. Users with visual impairments cannot see these links.

There are two ways to comply with this requirement:

1. You can make links appear differently from regular text without relying on color alone. For example, you can add an underline to your links. This makes links easy to identify, even to people who cannot see color at all.
2. You can make links a color which is very different from the surrounding text (a 3:1 ratio), AND ensure that when the link is selected with the keyboard or mouse that it appears
    differently from regular text without relying on color alone.

Unless you have disabled the outline CSS style, links will automatically appear visually distinct when selected with the keyboard.
